### ISSUE TITLE
As a user, I only want the to enter the first part of my email.

### ISSUE THREAD

**Issue by codingwithchris** [CreativeFuse/hancock#4]
The company's domain should already be populated & should be non-editable.

This could also mean that the company's domain should be a variable within Vue so a developer can set it easily.

**Comment by mrbeewer** [comment]
Could also have a variable that sets the company's standard name to email format? Such as `[firstNameFirstLetter + lastName]` ... but I guess there would be the occasional email that doesn't fit the standard form.

**Comment by codingwithchris** [comment]
@mrbeewer That's a good idea! The only issue we would have is that some people have to use their first and last name when we have more than 1 person with that name. So the format of the names themselves varies.

**Comment by mrbeewer** [comment]
@codingwithchris Yeah, that's what I was thinking towards the end with the occasional email that wouldn't fit. Probably wouldn't be a universally good option, unless the system was able to confirm that that email exists in the company's address list (so some form of verification) and change accordingly.

**Comment by codingwithchris** [comment]
It would be fun to really go overboard with this and do stuff like that!! Haha. Eventually, it could be built in as part of the hub where new users get a bunch of things sent to/created for them when they join the company! I wonder if Google has an API to auto-update signatures?!

**Comment by mrbeewer** [comment]
I believe that Google Suite API has a lot of things that could be helpful for the onboarding automation. With a little Googling, I found a mention of using the API to set a common signature for users, but didn't see anything about setting a specific one for each user (and not having them update it themselves). Would be a good add-on down the road.
